deploy/policy/templates/conjur-pet-store-secrets.template.sh.yml

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
its not part of demo only?

Reply:
i agree, looks like this file can be removed